schemas/vertices/gtdb/gtdb_taxon.json

```diff
@@ -1,23 +1,92 @@
 {
-  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$schema": "http://json-schema.org/draft-xxx/schema#",
   "type": "object",
-  "description": "A entry in the GTDB taxonomy tree.",
-  "required": ["_key", "type", "name"],
+  "description": "Template for a vertex entry in the NCBI or GTDB taxonomy tree.",
+  "required": ["_key", "source", "scientific_name", "rank"],
   "properties": {
     "_key": {
       "type": "string",
       "description": "Taxon type abbreviation plus name",
       "examples": ["d:Bacteria", "p:Firmicutes"]
     },
-    "name": {
+    "source": {
+      "type": "string",
+      "title": "Taxonomy source",
+      "examples": ["NCBI", "GTDB"]
+    },
+    "scientific_name": {
       "type": "string",
       "title": "Taxon name.",
-      "examples": ["Bacteria", "Firmicutes"]
+      "examples": ["Methylophilus methylotrophus", "Bacteria", "Firmicutes"]
+    },
+    "canonical_scientific_name": {
+      "type": "array",
+      "title": "Canonicalized scientific name",
+      "examples": [ [ "methylophilus", "methylotrophus" ], 
+                    ["Bacteria"], 
+                    ["Firmicutes"]
+                  ],
+      "items": { "type": "string" }
+    },
+    
+    "aliases": {
```

Comment:
I'm pretty sure gtdb won't have any aliases, so I think we'll want to remove this one